src/views/BookListViewAdminStore.tsx

```diff
@@ -13,7 +13,11 @@ const BookListViewAdminStore = () => {
       filter: '',
     },
     onSubmit: (values) => {
-      setFilter(values.filter);
+      if (!values.filter) {
+        setFilter('book');
+      } else {
+        setFilter(values.filter);
+      }
```

Comment:
You could've used an inline if there, like this:
```suggestion
      setFilter(!values.filter ? 'book' : values.filter);
```